gpiozero/tones.py

```diff
@@ -36,8 +39,17 @@ class to easily represent musical tones. The class can be constructed in a
 
     If you do not want the constructor to guess which format you are using
     (there is some ambiguity between frequencies and MIDI notes at the bottom
-    of the MIDI scale), you can use one of the explicit constructors,
-    :meth:`from_frequency`, :meth:`from_midi`, or :meth:`from_note`.
+    end of the frequencies, from 128Hz down), you can use one of the explicit
+    constructors, :meth:`from_frequency`, :meth:`from_midi`, or
+    :meth:`from_note`, or you can specify a keyword argument when
+    constructing::
+
+        >>> Tone.from_frequency(440.0)
+        >>> Tone.from_midi(69)
+        >>> Tone.from_note('A4')
+        >>> Tone(frequency=440.0)
```

Comment:
IIRC you said you were getting rid of the int-vs-float ambiguity, so I guess the `440.0`s here could just be `440`?

Reply:
Yup, certainly could be - I'll change it.